custom_conf.py

```diff
@@ -146,6 +146,7 @@
     'attachment:TranslationsSchema.dia',  # needs update
     r'https://github\.com/canonical/fetch-service*',  # private
     r'https://github\.com/canonical/fetch-operator*',  # private
+    'https://git.launchpad.net/charm-launchpad-buildd-image-modifier/tree/files/scripts/setup-ppa-buildd'  # private
```

Comment:
If you add a comma now, then when adding a new line, this line won't appear in the diff.